tests/streams/java/src/main/java/betterproto/CompatibilityTest.java

```diff
@@ -0,0 +1,32 @@
+package betterproto;
+
+import java.io.IOException;
+
+public class CompatibilityTest {
+    public static void main(String[] args) throws IOException {
+        if (args.length < 2) throw new RuntimeException("Attempted to run without the required arguments.");
```

Comment:
Should we also check `> 2` and emit an appropriate error message?

Reply:
Done